recipes/wbplot/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "wbplot" %}
+{% set version = "1.0.14" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: dac3c34077c4f351bfa5d61554374afce17fb6d39c9337f16c2fef633c49d660
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - matplotlib-base
+    - nibabel
+    - numpy
+    - pillow
+    - python
+
+test:
+  imports:
+    - wbplot
+    - wbplot.utils
```

Comment:
```suggestion
    - wbplot.utils
  commands:
    - pip check
  requires:
    - pip
```

Reply:
wbplot.util is used as a test case

Reply:
That is orthogonal to the `pip check` test. If you generate the recipes with grayskull the pip-check is added by default BTW.

Reply:
Understood